static/app/views/settings/components/teamSelect.tsx

```diff
@@ -92,6 +92,32 @@ function TeamSelect({
   const {teams, onSearch, fetching} = useTeams();
   const {orgRoleList, teamRoleList} = organization;
 
+  const slugsToFilter: string[] =
+    selectedTeams?.map(tm => tm.slug) || selectedTeamRoles?.map(tm => tm.teamSlug) || [];
+
+  // determine if adding a team changes the minimum team role
+  const getTopOrgRole = (): string | undefined => {
+    const teamsWithOrgRole = teams.filter(
+      team => slugsToFilter.includes(team.slug) && team.orgRole !== null
+    );
+    // get teams with org roles, if any
+    const allOrgRoles =
```

Comment:
maybe `allOrgRolesOnTeams`? or `...ForTeams`?
kinda weird since it's all the orgRoles for 'my' teams, not all existing org roles 🤔

Reply:
maybe `allMemberOrgRoles`